src/ngraph/frontend/onnx_import/core/graph.cpp

```diff
@@ -154,6 +154,19 @@ namespace ngraph
             }
             return results;
         }
+        const NodeVector& Graph::add_provenance_tags(const Node& onnx_node,
```

Comment:
I think this function shouldn't be a part of `Graph` class, since it isn't related in any way with it. This function is independent from `Graph` class. I would suggest to make it static or to move it to some file with helper functions.

Reply:
Ok, I moved it to the `onnx_import/utils/common`. Is it a better place for this function?